### PR TITLE
Frontend - WIP

### PR DESCRIPTION
**Pull request by biggiep2**

### CODE REVIEW

File: newamericadotorg/assets/react/surveys-homepage/components/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (biggiep2) [inline newamericadotorg/assets/react/surveys-homepage/components/.prettierrc]: Remove before merging

File: newamericadotorg/assets/react/surveys-homepage/components/BasicHeader.js [hunk 2]
```diff
@@ -3,16 +3,14 @@ import React from 'react';
...
-    .filter((item) => item !== 'new-america')
-    .join('/');
-  console.log(url);
+
   return (
     <div className="basic-header">
       <div className="basic-header__breadcrumb">
         <h6 className="link margin-0 with-caret--left">
-          <a href="/education-policy">Education Policy</a>
+          {/* <a href={data.survey_home_page.parent_project.url}>
```
Comment (biggiep2) [inline newamericadotorg/assets/react/surveys-homepage/components/BasicHeader.js]: Needs to pull from API

File: newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js [hunk 3]
```diff
@@ -266,7 +266,7 @@ class TeaserListing extends Component {
               key={`teaser-listing-item-${index}`}
               target="_blank"
             >
-              <div className="col-sm-9">
+              <div className="col-sm-9 col-md-10">
```
Comment (biggiep2) [inline newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js:269]: Need to confirm date does not break on all screen widths

File: newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js [hunk 4]
```diff
@@ -279,7 +279,7 @@ class TeaserListing extends Component {
                   />
                 )}
               </div>
-              <div className="col-sm-3 teaser-listing__item-date">
+              <div className="col-sm-3 col-md-2 teaser-listing__item-date">
```
Comment (biggiep2) [inline newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js:282]: Need to confirm date does not break on all screen widths

File: newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js (hunk 3 above)
Comment (LeeMellon) [reply newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js:269]: Everything seems to maintain continuity above 270px.

File: newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js (hunk 4 above)
Comment (LeeMellon) [reply newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js:282]: see above

**Pull request merged**